### ISSUE TITLE
repo is huge

### ISSUE THREAD

**Issue by chovy** [Diblo/KODI-Popcorn-Time#7]
Why is the repo over 100MB?

**Comment by Diblo** [comment]
Are you thinking about the branch "Repository" when it is extracted?

**Comment by chovy** [comment]
Just when I clone the repo, it is over 120MB. I'm guess you have something in your history that was quite large at one point.

**Comment by Diblo** [comment]
Yes, the reason to this is the files in the bin folder. They are 3th party software from https://github.com/steeve/libtorrent-go and https://github.com/steeve/torrent2http

**Comment by chovy** [comment]
There's no package install method? usually vendor libraries aren't committed to the repo, they are installed at build time.

**Comment by Diblo** [comment]
I know :+1:
I have not had time to check out the source code has been updated since the last release of xbmctorrent. So until advanced situated bin files in the repo. In this way it is easier to know what works.

But I has meanwhile searched an alternative three party solution and I found this plugin. I think that I will build my plugind on this provider in the future, but the bin files will remain until I finish the next version of KODI Popcorn Time :)

**Comment by chovy** [comment]
ok, sounds good. just be sure to delete the bins from the history as well. otherwise your repo will still be 121MB because they will be in the git history.

**Comment by Diblo** [comment]
:+1:
